server/routes/createGoal/CreateGoalController.ts

```diff
@@ -47,9 +51,27 @@ export default class CreateGoalController {
       const selectedAreaOfNeed = areasOfNeed.find(areaOfNeed => areaOfNeed.url === req.params.areaOfNeed)
       const minimumDatePickerDate = formatDateWithStyle(new Date().toISOString(), 'short')
 
+      const criminogenicNeedsData = req.services.sessionService.getCriminogenicNeeds()
       const planUuid = req.services.sessionService.getPlanUUID()
       const plan = await req.services.planService.getPlanByUuid(planUuid)
 
+      let assessmentDetailsForArea = null
+
+      // get assessment data
+      try {
+        const assessmentResponse = await req.services.assessmentService.getAssessmentByUuid(planUuid)
+        const assessmentData = assessmentResponse.sanAssessmentData
+
+        // 2. get the assessment details for the selected area of need
+        const areaConfig: AssessmentAreaConfig = areaConfigs.find(config => config.area === selectedAreaOfNeed?.name)
+        assessmentDetailsForArea = getAssessmentDetailsForArea(criminogenicNeedsData, areaConfig, assessmentData)
+
+        // todo: 6. write a cypress test for the view
+        // todo: 7. think about what to do for test data variations
+      } catch {
+        /* swallow the error, missing data is handled in the template */
+      }
```

Comment:
Have these todo's been actioned? Can they be removed?

Reply:
yep I will remove

Reply:
Removed in https://github.com/ministryofjustice/hmpps-sentence-plan-ui/pull/385/commits/86f254a792b3596aec53c6b341b0bdcb89f46c31